eg25-manager: remove

Luflosi opened the pull request:
eg25-manager is now in Nixpkgs. Without this change, the two versions collide and cause evaluation to fail.

At some point in the future, eg25-manager can be removed from Mobile NixOS.

I have not yet tested this PR on my PinePhone but it should still work.

Fixes #734.

Luflosi committed:
eg25-manager: remove

eg25-manager is now in Nixpkgs, so use that version instead.
Without this change, the two versions collide and cause evaluation to fail.

Luflosi commented:
After testing it, it seems this does not work properly yet. I'll try to fix it.

Luflosi commented:
Should be good now.

samueldr commented:
Let's

 1. remove it entirely
 2. noting that for anyone it breaks stuff that `nixos-unstable` is expected to be tracked
 3. update the pinned Nixpkgs version I guess, though I also have planned to just use `npins` instead of that ad-hoc manner. so it's not important to handle that for now.

Luflosi commented:
I now removed it entirely. CI is failing because we didn't update Nixpkgs yet. I'll look into how to do that tomorrow. Is there anything that needs to be done besides updating the revision and hash?

> noting that for anyone it breaks stuff that nixos-unstable is expected to be tracked

How to best do that? Write some code to detect when the option isn't present and then show an error message?

samueldr commented:
We can let it fail.

The README already states:

> ⚠️ Note: Mobile NixOS is only expected to build succesfully against the *unstable* branch of Nixpkgs.

Luflosi committed:
pkgs.nix: Update pinned Nixpkgs

Update it to the latest (at the time of making this commit) nixos-unstable from https://status.nixos.org/.

Luflosi commented:
I also updated Nixpkgs now. This should fix the CI failure.

Luflosi commented:
Thanks!
